docs/complete_configuration_guide.md

```diff
@@ -0,0 +1,353 @@
+open $HOME/.zatconfig/zat.plist
+```
+
+Your default text editor will open an empty file.
+Paste the below in the file.
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>RunAtLoad</key>
+   	<true/>
+	<key>Label</key>
+	<string>zat</string>
+	<key>ProgramArguments</key>
+	<array>
+		<string>/usr/local/bin/docker</string>
+		<string>run</string>
+		<string>--rm</string>
+		<string>-v</string>
+     <string>/Users/your-username-comes-here/.zatconfig:/app</string>
+		<string>docker.elastic.co/employees/nkammah/zat:latest</string>
+	</array>
+	<key>StandardOutPath</key>
+	<string>/tmp/zat.log</string>
+	<key>StandardErrorPath</key>
+	<string>/tmp/zat.log</string>
+	<key>StartCalendarInterval</key>
+	<array>
+		<dict>
+			<key>Hour</key>
+			<integer>18</integer>
+		</dict>
+	</array>
+</dict>
+</plist>
+```
+
+We'll need to modify 2 sections of the above file:
+1. Put the correct path to the zat configuration folder
+
+In your terminal type the below:
+```
+echo $HOME/.zatconfig
+```
+
+
+This will display the path to the .zatconfig file you need to insert and replace in the below line
+``` <string>/Users/your-username-comes-here/.zatconfig:/app</string>```
+
+
+2. Adjust the hour and times to run zat
+
+The below section defines when to run zat - in the below example, only once at 5:30pm
+```
+	<array>
+		<dict>
+			<key>Hour</key>
+			<integer>17</integer>
+			<key>Minute</key>
+			<integer>30</integer>
+		</dict>
+	</array>
+```
+
+And in the below example, every day at 2pm and every Thursday at 7pm.
+```
+	<array>
+		<dict>
+			<key>Hour</key>
+			<integer>14</integer>
+		</dict>
+		<dict>
+			<key>Weekday</key>
+			<integer>4</integer>
+			<key>Hour</key>
+			<integer>19</integer>
+		</dict>
+	</array>
+```
+
+You can specify StartCalendarInterval times using these time keys:
+
+```
+Minute <integer>
+The minute on which this job will be run.
+
+Hour <integer>
+The hour on which this job will be run.
+
+Day <integer>
+The day on which this job will be run.
+
+Weekday <integer>
+The weekday on which this job will be run (0 and 7 are Sunday).
+
+Month <integer>
+The month on which this job will be run.
+
+```
+
+
+### Load the configuration file
+
+In your terminal, type the below command:
+
+```
+launchctl load $HOME/.zatconfig/zat.plist
+```
+
+Eat will run the very first time - we'll check if it succeeded by reading the log file it generated.
+In your terminal, type the below command - the output should be similar than when you ran zat once alone.
+
+```
+tail /tmp/zat.log
+```
+
+
+### Modifying the zat schedule
+
+When you want to modify the schedule for zat, you'll need to:
+
+1. Remove the existing schedule
+```
+launchctl remove zat
+```
+
+2. Edit the new schedule
+3. Load it again
+```
+launchctl load $HOME/.zatconfig/zat.plist
+```
+4. Verify that it loaded correctly
+```
+tail /tmp/zat.log
+```
```

Comment:
Lots of overlap here with the README but these are excellent and I love the screenshots - I'd be happy to merge this in and consolidate the two docs in a follow up.